app/src/main/java/com/infomaniak/swisstransfer/ui/utils/Recaptcha.kt

```diff
@@ -0,0 +1,47 @@
+/*
+ * Infomaniak SwissTransfer - Android
+ * Copyright (C) 2024 Infomaniak Network SA
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package com.infomaniak.swisstransfer.ui.utils
+
+import android.app.Application
+import android.util.Log
+import com.google.android.recaptcha.Recaptcha
+import com.google.android.recaptcha.RecaptchaAction
+import com.google.android.recaptcha.RecaptchaClient
+import com.infomaniak.swisstransfer.BuildConfig
+import javax.inject.Inject
+
+class Recaptcha @Inject constructor(private val application: Application) {
+
+    private var client: RecaptchaClient? = null
+
+    suspend fun initializeClient() {
```

Comment:
A kDoc to explain that it is necessary to be called to the MainApplication, which may help us in the future.